OMEdit/OMEditGUI/Editors/MetaModelEditor.cpp

```diff
@@ -441,14 +441,14 @@ void MetaModelEditor::addInterfacesData(QDomElement interfaces)
         QDomElement interfacePoint;
         // update interface point
         if (existInterfaceData(subModel.attribute("Name"), interfaceDataElement)) {
-          interfacePoint = getInterfacePoint(subModel.attribute("Name"), interfaceDataElement.attribute("Name"));
-          interfacePoint.setAttribute("Name", interfaceDataElement.attribute("Name"));
+          interfacePoint = getInterfacePoint(subModel.attribute("Name"), interfaceDataElement.attribute("name"));
```

Comment:
I had updated the attribute in TLM code yesterday. So this code is invalid.
See here https://openmodelica.org/hg/TLMPlugin/rev/26bc3294a9c6